bad_code.py

```
@@ -0,0 +1,19 @@
+MIN_ITERATION_VALUE = 10
+MAX_ITERATION_VALUE = 5
+def do_something_crazy(x: int, y: int) -> int:
+    """
+    This function performs a series of operations based on the input values x and y.
+    It checks if x is greater than the threshold value and y is less than the max y value.
+    If the conditions are met, it iterates over a range of numbers and prints a message.
+    Finally, it returns the sum of x and y.
+    """
+    if x > THRESHOLD_VALUE:
+        if y < MAX_Y_VALUE:
+            for i in range(ITERATION_LIMIT):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)